Jenkinsfile

```diff
@@ -144,7 +144,6 @@ pipeline {
                     steps {
                         sh 'mkdir -p /home/ubuntu/bin/'
                         sh 'git clone --single-branch --branch ${KONG_SOURCE} https://github.com/Kong/kong.git ${KONG_SOURCE_LOCATION}'
-                        sh 'export RESTY_IMAGE_TAG=8 && make package-kong && make test && make cleanup'
```

Comment:
You can add the following to the overall docker file to fix the OpenSSL issues with `curl` and `wget`:

```
RUN sed -i'' 's/^mozilla\/DST_Root_CA_X3.crt$/!mozilla\/DST_Root_CA_X3.crt/g' /etc/ca-certificates.conf
RUN update-ca-certificates
```

See https://www.openssl.org/blog/blog/2021/09/13/LetsEncryptRootCertExpire/ for more details.